test/helpers/helpers.go

```diff
@@ -0,0 +1,99 @@
+package helpers
+
+import (
+	"testing"
+	"time"
+
+	volumesnapshotsv1beta1 "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const snapshotAPIVersion = "snapshot.storage.k8s.io/v1beta1"
+
+// CreateFakePV returns a fake PersistentVolume with the provided parameters
+func CreateFakePV(t *testing.T, name, driver, volume string, size int) *corev1.PersistentVolume {
```

Comment:
nit: do we really need a new package for this?

Reply:
Moved `helpers.go` into the `e2e` package.